src/web/templates/web/event_detail.html

```diff
@@ -15,7 +16,8 @@ <h1 class="mb-3 text-primary">
   </h1>
   
   <div class="mb-3 d-inline-block">
-    {% include 'spokanetech/partials/human_readable_datetime.htm' with object=object only %} for {{ object.duration }} hour{{ object.duration|pluralize }}
+    {% include 'spokanetech/partials/human_readable_datetime.htm' with object=object only %}
+    {% if object.duration %}for {{ object.duration|timedelta }}{% endif %}
   </div>
```

Comment:
I don't think this gets used when loading the event using Htmx. It has a different surrounding class, and is formatted weird. 
![image](https://github.com/SpokaneTech/SpokaneTech_Py/assets/17833726/5355392b-e288-4835-9e2d-391932bddf63)
![image](https://github.com/SpokaneTech/SpokaneTech_Py/assets/17833726/ac9db1bb-0c38-4e61-9e4b-76720c0ecfa6)


Navigating to the event using `http://localhost:8000/events/1` works & has the nice formatting, but its replaced when using the sidebar to view events.

Reply:
Thanks for pointing this out! I was only testing with the absolute URL. Fixed in this commit: https://github.com/SpokaneTech/SpokaneTech_Py/pull/71/commits/32a63d5cdc5d20cceab74caeb5d0909b2990e9c8